modules/exploits/windows/fileformat/documalis_pdf_editor_and_scanner.rb

```diff
@@ -0,0 +1,120 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::PDF
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'      => 'Documalis Free PDF ',
+      'Description' => %q{Documalis Free PDF Editor and Documalis Free PDF Scanner is prone to a security vulnerability when open PDF files.When the application is used to open a specially crafted PDF file, a buffer overflow occurs allowing arbitrary code execution.
+      },
+      'License'         => MSF_LICENSE,
+      'Author'          =>
+        [
+          'metacom', # Vulnerability discovery and PoC
+          '<metacom27[at]gmail.com>', # Metasploit module
+        ],
+      'References'      =>
+        [
+          ['EDB', ]
```

Comment:
Was there an EDB reference you wanted to add in here? This looks incomplete.

Reply:
Hmm I'm not seeing any entries on EDB for this so I'm going to replace this with a fake CVE entry and mark this as something to come back to once @metacom27 gets the proper CVE ID assigned to him.